src/utils/startPageAtTop.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
```

Comment:
Just as an example:
 
Before this commit if you were halfway down the landing page and clicked on 'Analyze Parts'
You would then be brought to the analysis page, but positioned in the middle of the page instead of at the top.

This really was only an issue on smaller screens, But it was a sloppy issue.